tests/test_rect2.h

```diff
@@ -288,6 +291,9 @@ TEST_CASE("[Rect2i] Basic getters") {
 	CHECK_MESSAGE(
 			rect.get_end() == Vector2i(1280, 820),
 			"get_end() should return the expected value.");
+	CHECK_MESSAGE(
+			rect.get_center() == Vector2i(640, 460),
+			"get_center() should return the expected value.");
```

Comment:
We should add another test case with an odd-numbered size.

Reply:
Updated tests with odd-numbered sizes for Rect2/Rect2i.
AABB test is already an odd numbered.